BUILD

```diff
@@ -38,17 +39,24 @@ cc_library(
         "core/string_utils.h",
         "core/unicode.h",
         "core/vm.h",
-        "include/libjsonnet.h",
+    ],
+    deps = [
+        "//include:libjsonnet",
```

Comment:
No worries. Yeah, `//include:libjsonnet` refers to the `:libjsonnet` target in `include/`.